packages/calypso-e2e/src/lib/test-account.ts

```diff
@@ -10,15 +11,25 @@ import {
 	getCalypsoURL,
 } from '../data-helper';
 import envVariables from '../env-variables';
+import { SecretsManager } from '../secrets';
 import { TOTPClient } from '../totp-client';
 import { LoginPage } from './pages/login-page';
 
+interface BearerTokenResponse {
+	success: string;
+	data: {
+		bearer_token: string;
+		token_links: string[];
+	};
+}
+
```

Comment:
The return type of the `.json()` method is [`unknown`](https://github.com/sindresorhus/got/blob/main/source/core/response.ts#L132) and so this interface is necessary.

Reply:
Oh, I would say this is common practice actually! A lot of these HTTP request libraries don't type the responses (or some take generic types, so you tell it what type you expect the response to be). This looks great! 😄